app/proxy/service.go

```diff
@@ -236,12 +252,12 @@ func (c *Caller) call(rawQuery []byte) (*jsonrpc.RPCResponse, CallError) {
 	}
 	execTime := time.Now().Sub(queryStartTime).Seconds()
 
-	c.service.LogExecTime(q.Method(), execTime, q.Params())
+	c.service.SetMetricsValue(q.Method(), execTime, q.Params())
 
 	if r.Error == nil {
-		c.service.logger.LogSuccessfulQuery(q.Method(), execTime, q.Params())
+		c.service.logger.LogSuccessfulQuery(q.Method(), execTime, q.paramsForLog())
```

Comment:
A better convention for go is to check if `r.Error != nil` and remove the else altogether.

Reply:
there is a non-returning pathway for when there is an error as well, so if/else is required. but yeah, replaced this with more familiar `!= nil`